helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -120,7 +120,7 @@ internal class HelperMain : CliktCommand(name = ORTH_NAME, epilog = "* denotes r
 fun CliktCommand.readOrtResult(ortFile: File): OrtResult {
     log.debug { "Input ORT result file has SHA-1 hash ${HashAlgorithm.SHA1.calculate(ortFile)}." }
 
-    val (ortResult, duration) = measureTimedValue { ortFile.readValue<OrtResult>() }
+    val (ortResult, duration) = measureTimedValue { ortFile.readValue<OrtResult>().withResolvedScopes() }
```

Comment:
Ah, here it comes. Then maybe the function docs should also only start mentioning it in this commit?